src/drive/web/modules/navigation/AppRoute.jsx

```diff
@@ -22,6 +22,16 @@ import UploadFromMobile from 'drive/mobile/modules/upload'
 import ExternalRedirect from './ExternalRedirect'
 import DriveView from '../views/Drive'
 
+// To keep in sync with AppRoute below, used to extract params
+// in the "router" redux slice. Innermost routes should be
+// first
+export const routes = [
```

Comment:
Mmh maybe but that would not be forward compatible with next versions of react-router right, I do not think you can use plain object configuration in newer versions since the routing is dynamic (no longer a central routing object). I think the keeping in sync is pretty straightforward/simple here and that there won't be so many route params that we want to keep in the router.  I suggest that we keep it like that for the moment.